model_analyzer/result/run_config_measurement.py

```diff
@@ -448,6 +448,34 @@ def compare_measurements(self, other: 'RunConfigMeasurement') -> float:
         # Step 3: Reverse the polarity to match what is expected in the docstring return
         return -1 * weighted_rcm_score
 
+    def compare_constraints(
+            self, other: 'RunConfigMeasurement') -> Union[float, None]:
+        """
+        Compares two RunConfigMeasurements based on how close
+        each RCM is to passing their constraints
+
+        Parameters
+        ----------
+        other: RunConfigMeasurement
+            
+        Returns
+        -------
+        float
+           Positive value if other is closer to passing constraints
+           Negative value is self is closer to passing constraints
+           Zero if they are equally close to passing constraints
+           None if either RCM is passing constraints
+        """
+        if self.is_passing_constraints() or other.is_passing_constraints():
+            return None
+
+        self_failing_pct = ConstraintManager.constraint_failure_percentage(
+            self._model_config_constraints, self)
+        other_failing_pct = ConstraintManager.constraint_failure_percentage(
+            other._model_config_constraints, other)
+
+        return (self_failing_pct - other_failing_pct) / 100
```

Comment:
What is the range of return values?

Reply:
0 to infinity, see my previous comment

But in practice, it will be 0.01 to a single digit number most of the time